src/controllers/helpers/followersHelper.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable require-jsdoc */
+
+import models from '../../models';
+
+const { User, Follow } = models;
+
+export default class FollowerHelper {
```

Comment:
These functions can be one-liner without async/await syntax and without being wrapped in a class